andino_hardware/README.md

```diff
@@ -25,6 +25,9 @@ This package aims to provide the necessary information to the correct assembly o
 
 <img src="docs/arduino_motor_diagram.png" />
 
+**Important❗  : As of a recent update, the A4 and A5 connections were moved to A2 and A3. This diagram will be updated soon. Refer to [andino_firmware/src/hw.h](../andino_firmware/src/hw.h) for pinout reference.**
```

Comment:
Would it make sense to point `recent update` to the PR?

Maybe something like: "recent update (#215)"

Reply:
It can be added, just in case anyone is curious about what caused this change.